src/components/i18n/FormattedCompMessage.js

```diff
@@ -150,7 +150,7 @@ class FormattedCompMessage extends React.Component<Props, State> {
     }
 
     render() {
-        const { count, tagName, intl, description, id, ...rest } = this.props;
+        const { count, tagName, intl, description, id, defaultMessage, ...rest } = this.props;
```

Comment:
It looks like `defaultMessage` is unused. Is that intentional?

Reply:
Yes, it was getting into the "rest" which made its way into the output. Anything that is not a prop we use directly should go onto the output span element like className for example.